src/jit/gentree.cpp

```diff
@@ -6386,6 +6407,13 @@ GenTreeArgList* Compiler::gtNewArgList(GenTreePtr arg1, GenTreePtr arg2)
     return new (this, GT_LIST) GenTreeArgList(arg1, gtNewArgList(arg2));
 }
 
+GenTreeArgList* Compiler::gtNewAggregate(GenTree* firstElement)
```

Comment:
Add a comment that says we are creating a special kind of GT_LIST that is used to hold multi-part struct aggregates.
You might also say that in the future we may change this GT_LIST into it's own node such as GT_AGGREGATE